ci/praktika/result.py

```diff
@@ -410,26 +411,38 @@ def create_new(cls, name, status, links=None, info="", results=None):
         )
 
     @classmethod
-    def from_gtest_run(cls, unit_tests_path, name="", with_log=False):
+    def from_gtest_run(
+        cls, unit_tests_path, name="", with_log=False, command_prefix=""
+    ):
         """
-        Runs gtest and generates praktika Result
-        :param unit_tests_path:
+        Runs gtest and generates praktika Result from results
+        :param unit_tests_path: path to gtest binary
         :param name: Should be set if executed as a job subtask with name @name.
         If it's a job itself job.name will be taken as name by default
-        :param with_log:
-        :return:
+        :param with_log: whether to log gtest output into separate file
+        :param command_prefix: prefix to add to gtest command
+        :return: Result
         """
+
+        command = f"{unit_tests_path} --gtest_output='json:{ResultTranslator.GTEST_RESULT_FILE}'"
+        if command_prefix:
+            command = f"{command_prefix} {command}"
+
         Shell.check(f"rm {ResultTranslator.GTEST_RESULT_FILE}")
         result = Result.from_commands_run(
             name=name,
             command=[
                 f"chmod +x {unit_tests_path}",
-                f"{unit_tests_path} --gtest_output='json:{ResultTranslator.GTEST_RESULT_FILE}'",
+                command,
             ],
-            with_log=with_log,
```

Comment:
Now we do not pass through `with_log` argument? Also why not enable it by default?